components/FlopAuctionBlock.js

```diff
@@ -194,6 +209,19 @@ const byTimestamp = (prev, next) => {
   return 0;
 };
 
+const checkUserBidStatus = (lastEvent, userAddress) => {
+  const { type , fromAddress } = lastEvent;
+  if (type !== 'Kick' && type !== 'Deal' && userAddress === fromAddress) {
+    return TOP_BIDDER;
+  }
+
+  if (type === 'Deal' && userAddress === fromAddress) {
```

Comment:
this is actually not right...more like:

```suggestion
const lastDent = "past event with type dent"
  if (lastEvent.type === 'Deal' && lastDent.userAddress === fromAddress) {
```

basically the `sender` of the `Deal` is not the winner, just the person to call `Deal`, the "winner" is the sender of the last `Dent`

Reply:
Crap.. that worked for me becase I called `Deal` manually